src/svc_vc.c

```diff
@@ -741,6 +761,119 @@ svc_vc_recv(SVCXPRT *xprt)
 		}
 
 		xd->sx_fbtbc = (int32_t)ntohl((long)xd->sx_fbtbc);
+
+		__warnx(TIRPC_DEBUG_FLAG_SVC_VC,
+			"sx_fbtbc = %08x", (int)xd->sx_fbtbc);
+
+		if (xd->sx_fbtbc == 0x0d0a0d0a) {
+			/* HA Proxy V2? */
+			uint32_t rest[2];
+			struct {
+			uint8_t ver_cmd;  /* protocol version and command */
```

Comment:
odd indentation?

Reply:
also, kind of looks like something almost part of portable inet headers...

Reply:
Again, included haproxy.h